Sources/Extensions/Swift/String/StringExtension.swift

```diff
@@ -53,11 +53,187 @@ extension String {
     }
 
     /// Check if the string contains an email email with a valid format.
+    @available(*, deprecated, message: "Use `validateEmailAddress()` instead")
     public var isEmail: Bool {
         let emailRegex = "[A-Z0-9a-z._%+-]+@[A-Za-z0-9.-]+\\.[A-Za-z]{2,6}"
         return NSPredicate(format: "SELF MATCHES %@", emailRegex).evaluate(with: self)
     }
 
+	/// These values are given in soft language intentionally because email validation is highly varied and many
+	/// places are too strict. The support level is based on authors anecdotal experiences.
```

Comment:
```suggestion
    /// These values are given in soft language intentionally because email validation is highly varied and many
    /// places are too strict. The support level is based on authors anecdotal experiences.
```